api/desecapi/tests/test_rrsets_bulk.py

```diff
@@ -48,14 +48,25 @@ def test_bulk_post_my_rr_sets(self):
         self.assertRRSetsCount(response.data, self.data)
 
     def test_bulk_post_rr_sets_empty_records(self):
-        # TODO discuss if we should instead of given an error just discard empty-record rr sets
+        with self.assertPdnsRequests(self.requests_desec_rr_sets_update(self.my_empty_domain.name)):
+            expected_response_data = [copy.deepcopy(self.data_empty_records[0]), None]
```

Comment:
This is the expected behavior for `PATCH` (and putentially `PUT`). For `POST`, non-empty `records` are always required (create operation).

`PATCH` should accept `records == []` regardless of whether the RRset is known or not.